.github/workflows/ci.yml

```
@@ -0,0 +1,52 @@
+    - name: Run code formatter check (Black)
+      run: |
+        poetry run black --check .
+
+    - name: Run import sorting check (isort)
+      run: |
+        poetry run isort --check .
+
+    # 6. 테스트 실행
+    - name: Run tests with Pytest
+      run: |
+        poetry run pytest
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)